content/logs/graph.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Log Graph
+kind: documentation
+description: "Perform analytics with Log Graphs"
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+---
+
+Switch between the Log List and the Log Graph mode by clicking on this button:
+
+{{< img src="logs/graph/log_graph_switch.png" alt="Log graph switch" responsive="true" popup="true" style="width:40%;">}}
+
+## Log graph query 
+
+Use the query to control what's displayed in your log graph:
```

Comment:
"Log Graph" is capitalised above, but lower-case here. Is that correct? (I don't know!)

Reply:
it should be capitalised indeed :)